client/src/ui/track/Searchbox.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Autocomplete from "@mui/material/Autocomplete";
+import { TextFieldProps, TextField, Typography } from "@mui/material";
+
+/**
+ * A custom version of the Autocomplete component with the label designed similarly to the InputField
+ * @param props TextFieldProps
+ */
+export function ComboBox(props: TextFieldProps) {
+  const { required, label, placeholder } = props; // Extracting placeholder
+  const { style, ...restProps } = props;
+
+  return (
+    <div style={style}>
+      <div
+        style={{
+          display: "flex",
+        }}
+      >
+        <Typography
+          sx={{
+            textAlign: "start",
+            fontWeight: "bold",
+            display: "flex",
+          }}
+        >
+          {label}
+        </Typography>
+        {required && (
+          <Typography
+            sx={{
+              paddingLeft: "5px",
+              color: "red",
+            }}
+          >
+            *
+          </Typography>
+        )}
+      </div>
+
+      {/* Autocomplete with TextField input */}
+      <Autocomplete
+        disablePortal
+        options={[]} // Will fix this later
+        PaperComponent={(props) => (
+          <div
+            {...props}
+            style={{
+              boxShadow: "none", 
+              border: "1px solid rgba(0, 0, 0, 0.12)", 
+              borderRadius: "8px", 
+              backgroundColor: "white", 
+            }}
+          />
+        )}        
+        renderInput={(params) => (
```

Comment:
Why not just use the custom InputField component here?

Reply:
Fixed!